toolkit/themes/shared/design-system/config.json

```diff
@@ -1,12 +1,35 @@
 {
-  "source": ["design-tokens.json"],
+  "source": ["test-tokens.json"],
```

Comment:
Instead of having a `test-tokens.json` file and referencing it here, let's modify `design-tokens.json` to use the format you need as part of this patch